django/contrib/auth/tokens.py

```diff
@@ -11,6 +11,9 @@ class PasswordResetTokenGenerator:
     reset mechanism.
     """
     key_salt = "django.contrib.auth.tokens.PasswordResetTokenGenerator"
+    algorithm = "sha256"
+    # Can be set to None when we are no longer in an algorithm transition period.
+    legacy_algorithm = "sha1"
```

Comment:
I don't think that we would like to keep handling of `legacy_algorithm` when deprecation ends, that's why I'm going to simplify this. I hope you don't mind.

Reply:
I have been thinking about this too, what happens if we were to deprecate sha256 at some point again? Do we want to leave legacy_algorithm around for that? (Porbably no)

Reply:
> Do we want to leave legacy_algorithm around for that? (Porbably no)

I don't think so.